lib/node_modules/@stdlib/blas/ext/base/dsumors/manifest.json

```diff
@@ -34,7 +34,12 @@
 				"-lm"
```

Comment:
Please change indentation of the `manifest.json` to 2-space and add configurations for benchmarks and examples. See the linked changes in the RFC issue or some of the recently merged PRs for how this will look.